pages/a/decred-101.vue

```diff
@@ -303,14 +335,15 @@
                 rules, they set a course for future development together, and
                 can make use of a development fund which receives 10% of each
                 block's reward. Decred’s blockchain is similar to Bitcoin’s, but
-                with major improvements of transparent and inclusive
+                with major improvements to the funding model as well as transparent 
+                and inclusive
                 <a
                   href="https://docs.decred.org/governance/overview/"
                   target="_blank"
                   class="underline"
                   >governance</a
                 >
-                baked into the protocol.
+                baked into the protocol. Coinholders have a say in Decred unlike Bitcoin. 
```

Comment:
This sounds a bit rough, I'm sure we can find a better phrasing. Eg. 

`If you hold Bitcoin, but you don't like how the protocol is evolving (or is not evolving), you can't do much about it. In Decred, people owning Decred can actively participate in "governance" and actively steer the path of the project.`